include/dice/template-library/polymorphic_allocator.hpp

```diff
@@ -248,7 +248,7 @@ namespace dice::template_library {
 
 		template<typename U>
 		constexpr polymorphic_allocator(polymorphic_allocator<U, Allocator> const &other) noexcept(std::is_nothrow_constructible_v<Allocator<T>, Allocator<U> const &>)
-			: alloc_{other} {
+			: alloc_{other.alloc_} {
```

Comment:
good find

We should probably start covering all ctors that are not `=default` or that use such a constructor that is not used elsewhere in least one test. Coverage reports would help here.

Reply:
only found it because I used this branch in another library and it didn't compile.
So yeah maybe we should, thats really annoying tho 😅